Ignore division if missing

Jassob opened the pull request:
Not all reports specify a division and this should therefore not be a
required field by the template.

Omitting division previosly caused LaTeX fail with inexplicable
errors--it attempts to end a line and place some vertical space, but
unless the division exists the line is empty and TeX becomes sad.
This commit fixes this by not attempting to typeset division if omitted.

Jassob committed:
Ignore division if missing

Not all reports specify a division and this should therefore not be a
required field by the template.

Omitting division previosly caused LaTeX fail with inexplicable
errors--it attempts to end a line and place some vertical space, but
unless the division exists the line is empty and TeX becomes sad.
This commit fixes this by not attempting to typeset division if omitted.

ayberkt commented on frontmatter/titlepage.tex:
```diff
@@ -77,8 +77,9 @@
 	\end{figure}	\vspace{5mm}
 
 	Department of Computer Science and Engineering\\
+        \ifdefined\thedivision
```
Is this indentation intended?

Jassob replied:
Huh no, it is not... Good catch!

Jassob replied:
Ugh, it's a tabs-vs-spaces thing...
For some reason my editor does not respect the original tabs and instead indents it with spaces..

ayberkt replied:
What? Are these files indented with tabs :angry:? I've never noticed that. I hate tabs and propose that we get rid of all of them.

Jassob committed:
Fix indentation in titlepage.tex

We managed to mix tabs and spaces in commit
6354bd079bac0a5141e1aaa0bce75ee7989c11d0. This commit converts those
spaces to tabs to keep indentation uniform.

Jassob commented:
@ayberkt I've fixed the indentation now (also added a commit to fix the indentation regarding subtitles). Please re-review :)
